### ISSUE TITLE
Some remarks

### ISSUE THREAD

**Issue by polarapfel** [woganmay/gdpr-blackhole#1]
Hi there,

this could be very useful. Just a few remarks:

1. It would help to reveal from where these IP address ranges were pulled.

2. Typically, IP address ranges change, countries gain or lose ranges from time to time. Hence, any list of IP address ranges per country must be updated in a regular fashion.

3. Based on the first two remarks, this repository is only really useful if it's transparent where these addresses came from and if the repository pulls regular updates from that source.

I'll be using https://www.ip2location.com/free/visitor-blocker manually for now.

**Comment by woganmay** [comment]
Those are good remarks! I'll update the documentation to include the source. They do offer the text files via a set of public endpoints, so an update script is definitely doable.

**Comment by woganmay** [comment]
Yeah, I'm aware of the scope of GDPR, which is why I added this to the readme upfront:

> It's not bullet-proof, of course: EU residents can travel and use VPNs, and there's no guarantee that simply blocking the IP ranges of the EU is a perfect shield.
> It affords no protections against GDPR enforcement, and is not a substitute for legal advice.

This repo is really just a convenience for anyone that needs a quick download of all the EU IP ranges, in the event they receive a short-sighted mandate from above to rapidly block the entire set. It's absolutely not the approach I would take for any of my own projects, and it's not something I'd recommend to anyone I work with - but that doesn't mean that everyone in the world is going to see things the same way :)
